From 8d494572db7cf0112ecd6bde2cb89b8c336ad69f Mon Sep 17 00:00:00 2001
From: Konstantin Porotchkin <kostap@marvell.com>
Date: Mon, 13 Aug 2018 15:05:18 +0300
Subject: [PATCH 0183/1239] arm64: mvebu: config: increase the malloc size to
 16MB

The boot on some Marvell platforms is unstable and may freeze during
network initialization due to insufficient allocation space for RX/TX
buffers.
This patch increases the CONFIG_SYS_MALLOC_LEN from 8MB to 16MB for
all Armada 64-bit plaforms for improving the boot stability.

Change-Id: I26ca91f47710e8bdf632f986300841b8535fdfa0
Signed-off-by: Konstantin Porotchkin <kostap@marvell.com>
---
 include/configs/mvebu_armada-common.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/mvebu_armada-common.h b/include/configs/mvebu_armada-common.h
index 97ee5fdd04..181359f12f 100644
--- a/include/configs/mvebu_armada-common.h
+++ b/include/configs/mvebu_armada-common.h
@@ -85,7 +85,7 @@
 /*
  * Size of malloc() pool
  */
-#define CONFIG_SYS_MALLOC_LEN	(8 << 20) /* 8MiB for malloc() */
+#define CONFIG_SYS_MALLOC_LEN	(9 << 20) /* 16MiB for malloc() */
 
 /*
  * Other required minimal configurations
-- 
2.29.0

